packages/remix/src/client/uiComponents.tsx

```diff
@@ -0,0 +1,33 @@
+import { SignIn as BaseSignIn, SignUp as BaseSignUp } from '@clerk/clerk-react';
+import type { SignInProps, SignUpProps } from '@clerk/types';
+import React from 'react';
+
+import { useClerkRemixOptions } from './RemixOptionsContext';
+
+export const SignIn = (props: SignInProps) => {
+  const { signInUrl } = useClerkRemixOptions();
```

Comment:
@anagstef @desiprisg 
Can you please remind me why we need a context for these options in Remix and Nextjs?
Why can we not simply pass the options to ClerkProvider and handle this internally within clerk-js?